datafu.yml

```diff
@@ -0,0 +1,16 @@
+name: Apache DataFu
+desc: DataFu is a collection of libraries for working with large-scale data in Hadoop.
+site: http://datafu.apache.org/
+tags:
+- scala
+- apache
+- pig
+- java
+- python
+- spark
+- hadoop
+- python
+- hive
+upforgrabs:
+  name: up for grabs
+  link: https://issues.apache.org/jira/issues/?jql=project%20%3D%20datafu%20AND%20status%20%3D%20Open%20AND%20labels%20%3D%20up-for-grabs%20ORDER%20BY%20created%20DESC
```

Comment:
@eyala This seems good too but we generate projects stats. So how about using `https://github.com/apache/datafu/labels/up%20for%20grabs`?
You will have to create a label `up for grabs` under that Github repo first.

Reply:
The thing is, we put bugs and small feature requests in our JIRA, not on GitHub. Is it possible to add more than one link?

Reply:
Nope, in that case I would keep this link. But then you won't be able to get projects stats on the site like other projects.